src/Symfony/Component/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php

```diff
@@ -52,7 +52,7 @@ public function handleRequest(FormInterface $form, $request = null)
         }
 
         // For request methods that must not have a request body we fetch data
-        // from the query string. Otherwise we look for data in the request body.
+        // from the query string. Otherwise, we look for data in the request body.
```

Comment:
CS changes should be submitted separately on the lowest branch where they apply (and/or be reverted from this PR)